Tests/Algorithm/AlgorithmAddDataTests.cs

```diff
@@ -100,6 +103,35 @@ public void CustomDataTypes_AreAddedToSubscriptions_Successfully()
             Assert.AreEqual(quandlSubscription.Type, typeof(Quandl));
         }
 
+        [Test, Ignore]
+        public void PythonCustomDataTypes_AreAddedToSubscriptions_Successfully()
+        {
+            var pythonPath = new System.IO.DirectoryInfo("RegressionAlgorithms");
+            Environment.SetEnvironmentVariable("PYTHONPATH", pythonPath.FullName);
+
+            using (Py.GIL())
+            {
+                var module = Py.Import("Test_CustomDataAlgorithm");
+                var qcAlgorithm = new AlgorithmPythonWrapper(module);
+
+                // Initialize contains the statements:
+                // self.AddData(Nifty, "NIFTY")
+                // self.AddData(QuandlFuture, "SCF/CME_CL1_ON", Resolution.Daily)
+                qcAlgorithm.Initialize();
+
+                var niftySubscription = qcAlgorithm.SubscriptionManager.Subscriptions.FirstOrDefault(x => x.Type.Name == "Nifty");
+                Assert.AreEqual(niftySubscription.Type.Name, "Nifty");
```

Comment:
I think more correctly this should be `Assert.IsNotNull(niftySubscription)`, since `FirstOrDefault` will return null if not found. Also, I try to *never* use the string type names for comparisons unless absolutely necessary. If possible, update these to check the actual type using `x.Type == typeof(Nifty)` syntax.

Reply:
It is not possible to use `x.Type == typeof(Nifty)`, because `Nifty` is not a C#, it doesn't exist in QuantConnect Assembly. So maybe we could just we could just use `Assert.IsNotNull(niftySubscription)` and use `x.Symbol.Value == "NIFTY"` instead.